test/e2e/nsx_security_policy_test.go

```diff
@@ -84,12 +82,10 @@ func testSecurityPolicyBasicTraffic(t *testing.T) {
 	assert.NoError(t, testData.waitForResourceExistOrNot(ns, common.ResourceTypeRule, ruleName0, true))
 	assert.NoError(t, testData.waitForResourceExistOrNot(ns, common.ResourceTypeRule, ruleName1, true))
 
-	// Temporarily disable traffic check
-	/*
-		// Ping from pod
-		err = testData.runPingCommandFromPod(ns, busybox, iPs, 4)
-		assertNotNil(t, err, "Error when running ping command from test Pod %s", busybox)
-	*/
+	// Test traffic from client Pod to server Pod
+	trafficErr = assureTraffic(ns, clientPodName, containerName, iPs.ipv4.String(), podPort, 1*time.Second, 5*time.Second)
```

Comment:
Maybe you can define constants or global variables for traffic verification interval and timeout?

Reply:
ok